test/util/handleChangeDate.spec.js

```diff
@@ -0,0 +1,19 @@
+import handleChangeDate from '../../src/util/handleChangeDate';
+
+const dateFormat = 'DD/MM/YYYY HH:mm';
+const date = new Date('1995-12-17T03:24:00');
+
+describe('handleChangeDate', () => {
+    it('Should change Date', () => {
```

Comment:
rename to `should change a valid date`, pero te falta tests para casos inválidos.